js/controllers/SiteHandler.js

```diff
@@ -459,6 +460,22 @@
 	};
 
 
+	/**
+	 * Reload a tab.
+	 * @private
+	 * @param {HTMLElement} sourceElement The element that issued the
+	 *  "reloadTab" event.
+	 * @param {Event} event The "reload tab" event.
+	 * @param {Object} jsonData The JSON content representing the
+	 *  reload request.
+	 */
+	$.pkp.controllers.SiteHandler.prototype.reloadTabHandler_ =
+			function(sourceElement, event, jsonData) {
+
+		$(jsonData.tabsLocator).tabs('load', jsonData.tabLocator);
```

Comment:
I'm being picky, but can we change this from `tabLocator` to `tabSelector`? I think the term `selector` will be more easily recognized by CSS/JS devs.

Reply:
Of course -- that was a thinko!